fedimint-core/src/fee_consensus.rs

```diff
@@ -31,6 +31,35 @@ impl FeeConsensus {
         })
     }
 
+    /// The mint module will charge a non-configurable base fee of one hundred
+    /// millisatoshis per transaction input and output too account for the costs
```

Comment:
`s/too/to/`

Reply:
what?

Reply:
Sorry, that's `sed`/`vim` syntax for replacing `too` with `to`. I think that would be the grammatically correct way to do it.

Reply:
Done